docs/source/checks/nlp/train_test_validation/plot_property_drift.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+"""
+.. _nlp__property_drift:
+
+NLP Property Drift
+******************
+
+This notebooks provides an overview for using and understanding the nlp property drift check.
+
+**Structure:**
+
+* `Calculating Drift for Text Data <#calculating-drift-for-text-data>`__
+* `Prepare data <#prepare-data>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+* `Check Parameters <#check-parameters>`__
+
+Calculating Drift for Text Data
+=================================
+
+What is Drift?
+----------------
+
+Drift is simply a change in the distribution of data over time,
+and it is also one of the top reasons why machine learning model's performance degrades over time.
+
+For more information on drift, please visit our :doc:`drift guide </user-guide/general/drift_guide>`.
+
+How Deepchecks Detects Drift in NLP Data
+-----------------------------------------
+
+This check detects drift by in NLP Data by calculating various properties of the text and writing them into a table,
+where each sample has one row per text sample and one column per textual property. Then, drift can be calculated using
+:ref:`univariate measures <drift_detection_by_univariate_measure>` on each text property separately.
+
+Which NLP Properties Are Used?
+-------------------------------
+
+The properties used by this check are the ones present in datasets prior to running this check. That's why it's
+important to first run ``calculate_default_properties`` on both the train and test datasets prior to running this
+check. Note also that in order to calculate drift on more complex properties, such as text Toxicity or Fluency,
+you need to pass the ``include_long_calculation_properties=True`` parameter to ``calculate_default_properties``. For
+more information on how to calculate properties, please visit the :doc:`nlp properties guide
+</user-guide/nlp/nlp_properties>`.
+
```

Comment:
If you're missing some "content" at the beginning of the file, I would suggest adding some explanations on WHY we're looking for drift in the properties (not being able to check on the text itself).

Reply:
added something